src/collective/ifttt/actions/actions.zcml

```diff
@@ -6,12 +6,12 @@
   <configure zcml:condition="installed collective.easyform">
 
       <utility
-        name="collective.ifttt.actions.easyform"
+        name="collective.ifttt.actions.easyform.IFTTTTrigger"
         component=".easyform.IFTTTAction"
         />
 
       <utility
-        name="collective.ifttt.actions.easyform"
+        name="collective.ifttt.actions.easyform.IFTTTTrigger"
```

Comment:
@Shriyanshagro I believe this was you issue with not being able to add IFTTT Trigger easyform action.

I don't recall why utility name is required to match its class name here. It could be a convention for plone.supermodel somewhere in its README or just some custom convention in easyform.

(plone.supermodel provides XML serialization for zope.schema-based schemas and easyform uses that to store the form configuration to make it easy to export and import forms [e.g. to reuse and share them]).

Reply:
Thanks that works!! :)